pkg/api/kptfile/v1/types.go

```diff
@@ -12,7 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// Package defines the schema for Kptfile version v1.
+// Package defines Kptfile schema.
+// Version: v1
+// swagger:meta
```

Comment:
Did we checkout using `controller-gen` tool from kubebuilder project ? 
One good thing about `controller-gen` is that it generates the schema as well as CRD definition as well (which probably is required by cloudcode ?) @phanimarupaka can confirm.
Can we confirm if we use this schema with cloudcode, we get authoring assistance in cloudcode ?

Reply:
Updating this as per offline discussion. So we know cloudcode supports normal openapi schema along with the CRDs, so we are good.
And you also found out that controller-gen requires embedding the metadata from apimachinery. 

I think it's fine to move ahead with go-swagger for now.